### ISSUE TITLE
kivy.core.audio.audio_sdl2 uses deprecated property of kivy.core.audio.Sound

### ISSUE THREAD

**Issue by pydsigner** [kivy/kivy#7164]
**Software Versions**
* Python: 3.7.1
* OS: Windows 10
* Kivy: 1.11.1
* Kivy installation method: pip

**Describe the bug**

The `audio_sdl2` Audio provider uses the deprecated property `filename` of `kivy.core.audio.Sound`, causing deprecation warnings to be printed to the console.

**Expected behavior**
Loading sounds using the `audio_sdl2` provider should run without warnings being produced by code within kivy.

**To Reproduce**
```py
from kivy.core.audio import SoundLoader
SoundLoader.load(r'C:\Windows\Media\chimes.wav')
```

**Comment by pydsigner** [comment]
My apologies, I was somehow looking at an outdated version of kivy on Github to confirm the issue remained.

**Comment by pydsigner** [comment]
Fixed in 5acbdc4b00919f0b725a57d52f196cd609c87413

**Comment by tshirtman** [comment]
Ah, well, thanks anyway, don't hesitate contribute again if you see other things to fix <3.
